### PR TITLE
Update the file

### PR DESCRIPTION
**Pull request by dnellutla**
Should the line 213 and 216 be AuthenticationEndpointUtil instead of IdentityManagementEndpointUtil

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/develop/customizing-identity-server-uis.md [hunk 1]
```diff
@@ -210,10 +210,10 @@ Add the `product-footer.jsp` and `header.jsp` files to the extensions folder as
...
                 <p>XYZ Company | &copy;
                     <script>document.write(new Date().getFullYear());</script>
-                    <a href="<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "business.homepage")%>"
+                    <a href="<%=AuthenticationEndpointUtil.i18n(resourceBundle, "business.homepage")%>"
                        target="_blank">
                        <i class="icon fw fw-wso2"></i>
-                       <%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "Inc")%>
+                       <%=AuthenticationEndpointUtil.i18n(resourceBundle, "Inc")%>
                     </a>
                     . <%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "All.rights.reserved")%>
```
Comment (IsuraD) [inline en/docs/develop/customizing-identity-server-uis.md:218]: Don't we need to change this one too? 

                    . <%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "All.rights.reserved")%>

Comment (nilminiwso2) [reply en/docs/develop/customizing-identity-server-uis.md:218]: @brionmario @thivi 
Any thoughts on this?

Comment (dnellutla) [reply en/docs/develop/customizing-identity-server-uis.md:218]: @IsuraD I think the All Rights Reserved needs to be changed to ..

Comment (IsuraD) [reply en/docs/develop/customizing-identity-server-uis.md:218]: YEs. Change it

**Comment by madurangasiriwardena** [comment]
@dnellutla any update on the requested change?

File: en/docs/develop/customizing-identity-server-uis.md (hunk 1 above)
Comment (dnellutla) [reply en/docs/develop/customizing-identity-server-uis.md]: Done.

**Comment by dnellutla** [comment]
@madurangasiriwardena @IsuraD  Updated the tag.

**Review by IsuraD — APPROVED** [review]

**Pull request merged**